MCDataProducts/inc/SimParticle.hh

```diff
@@ -284,6 +294,8 @@ namespace mu2e {
     XYZTVectorF                    _endMomentum;
     float                  _endGlobalTime;
     float                  _endProperTime;
+    // float                  _preLastStepGlobalTime;
```

Comment:
and here

Reply:
Will do. I was going to remove them and tidy up the white space in a separate commit.

Reply:
Can you turn that around quickly tomorrow?

Reply:
Done